migrations/20231025145134_add_notification.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TYPE notification_states
+  AS ENUM ('queued', 'processing', 'published', 'not-subscribed', 'wrong-scope', 'rate-limited');
+
+CREATE TABLE notification (
+    id         UUID         PRIMARY KEY DEFAULT gen_random_uuid(),
+    created_at TIMESTAMPTZ  NOT NULL DEFAULT now(),
+    type       VARCHAR(255) NOT NULL,
+    title      VARCHAR(255) NOT NULL,
+    body       VARCHAR(255) NOT NULL,
+    icon       VARCHAR(255) NOT NULL,
+    url        VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE notification_delivery (
```

Comment:
Can we rename this to `subscriber_notification` since it's representing the per-subscriber notification? We don't actually know if it was delivered yet. That would be another set of specs we have to work on for receiving delivery messages which would further progress the status field in `subscriber_notification`

Reply:
Makes sense. Yes, we can change it (I'll change the name).

Reply:
Done.